### ISSUE TITLE
2.0 --> 2.1 socket bonus changes requirement is not refreshed

### ISSUE THREAD

**Issue by Mikik3jr** [Atlantiss/NetherwingBugtracker#5903]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 
As you can see on this picture I have 1993 bonus healing before regemming my belt for socket bonus.
![WoWScrnShot_061819_180608](https://user-images.githubusercontent.com/43505031/59703812-06e4fd80-91fb-11e9-93bf-af0f68c492f1.jpg)

**Current behaviour**:
I had an 8 intellect and 18 bonus healing gem in [my belt](https://tbc-twinhead.twinstar.cz/?item=30030)  (because in 2.0 that was the requirement for the 7bh set bonus), but in 2.1 I require 2 yellow instead of 1 yellow 1 red to get the socket bonus. I equiped a 9bh + 4 intellect into my belt, but my bh dropped from 1993-->1986.
![image](https://user-images.githubusercontent.com/43505031/59704004-6a6f2b00-91fb-11e9-9622-1eea72afacbc.png)

With the 9bh +4 intellect +7 socket bonus I should get 16 (+1.5 from talent from intellect) vs. 18bh red gem, so the difference should be 0 or 1, not 7 (which is exactly the bonus healing from socket bonus).

**TL;DR: My belt needed 1 red 1 yellow gem for socket bonus, after 2.1 patch I need 2 yellow gems, but I still had the socket bonus with 1 red and 1 yellow gem.**

**Expected behaviour**: 
I should not get my socket bonus, if the requirement changed in 2.1 for it.

**Server Revision**: 
2014

**Comment by Manaci** [comment]
Irrelevant now

**Comment by Manaci** [comment]
Reopened for future usage

**Comment by Godmatik** [comment]
2.0 stats will no longer be used, closing
